### ISSUE TITLE
Update samples to use SecurityFilterChain bean instead of WebSecurityConfigurerAdapter

### ISSUE THREAD

**Issue by eleftherias** [spring-projects/spring-security#8821]
Depends on: gh-8805

**Comment by rwinch** [comment]
We should also hold off on the sample rewrites that I'm doing right now

**Comment by eleftherias** [comment]
Closing as duplicate of https://github.com/spring-projects/spring-security-samples/issues/52 which is complete.
